README.md

```diff
@@ -1 +1,75 @@
-# Private-Events
\ No newline at end of file
+<h3 align="center">Private Events - RoR</h3>
+
```

Comment:
## Add a screenshot of your page on the README 

- [ ] Add a screenshot of your page here. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)).